absl/synchronization/internal/kernel_timeout.h

```diff
@@ -117,9 +114,11 @@ class KernelTimeout {
   // Windows. Callers should recognize that the return value is a
   // relative duration (it should be recomputed by calling this method
   // in the case of a spurious wakeup).
-  DWORD InMillisecondsFromNow() const {
+  typedef unsigned long dword;
```

Comment:
Type names start with a capital letter and have a capital letter for each new word, with no underscores: MyExcitingClass, MyExcitingEnum.

Reply:
Changed it to `DWord`. As far as I know, it stands for "double word".

https://docs.microsoft.com/en-us/windows/desktop/winprog/windows-data-types